torchaudio/transforms/_transforms.py

```diff
@@ -1566,10 +1579,11 @@ def forward(self, waveform: Tensor) -> Tensor:
         Returns:
             Tensor: The pitch-shifted audio of shape `(..., time)`.
         """
+        # retrieving shape
```

Comment:
I would remove the comment and move `shape = waveform.size()` to above the post process function, right before it is used

Reply:
edit: waveform shape is temporarily modified in the preprocess function, so we need to move it back